pkg/ring/kv/consul/metrics.go

```diff
@@ -1,82 +0,0 @@
-package consul
-
-import (
-	"context"
-
-	consul "github.com/hashicorp/consul/api"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/weaveworks/common/instrument"
-)
-
-var consulRequestDuration = instrument.NewHistogramCollector(prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Please remember to mention in the CHANGELOG that the metric `cortex_consul_request_duration_seconds` has been removed.

Reply:
I re-added this metric since it fits a different use case.